homework_9

IsabellaDu opened the pull request.

IsabellaDu committed:
add package homework_9

Vladyslavtovkun commented on src/my_package_name/homework_9/Student.java:
```diff
@@ -0,0 +1,27 @@
+    String name;
+    int course;
+
+    public String getName() {
```
Видалить будь-ласка код який не використовується

Vladyslavtovkun commented on src/my_package_name/homework_9/CollectionOfStudents.java:
```diff
@@ -0,0 +1,34 @@
+        return students;
+    }
+
+    public static ArrayList<Student> addNewStudent(ArrayList<Student> listOfStudents, Student newStudent) {
```
не розумію навіщо в методах addStudent, removeStudent треба щось повертати

Vladyslavtovkun commented on src/my_package_name/homework_9/CollectionOfStudents.java:
```diff
@@ -0,0 +1,34 @@
+
+public class CollectionOfStudents {
+
+    public static ArrayList<Student> generateListOfStudents() {
```
Краще заповнювати колекцію в мейні, для більшої наглядності. + перейменувати класс, наприклад StudentsMethodHelper, бо колишня назва втратить своє значення

IsabellaDu committed:
rename class CollectionOfStudents to StudentsMethodHelper
change methods addStudent, removeStudent to void methods
remove methods from Student.java
